From 0415d0e5f8266ff67dfd8204ee4fb1ad0ce1d5cf Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Tue, 1 Dec 2020 14:52:41 -0800
Subject: [PATCH] Re-fix Ghana typo for 1919/1920

* africa (Ghana): Re-fix the typo in previous change to these rules.
(Thanks to P Chan.)
---
 africa | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/africa b/africa
index 85644f5..429f783 100644
--- a/africa
+++ b/africa
@@ -457,7 +457,7 @@ Zone	Africa/Cairo	2:05:09 -	LMT	1900 Oct
 # Rule	NAME	FROM	TO	-	IN	ON	AT	SAVE	LETTER/S
 Rule	Ghana	1919	only	-	Nov	24	0:00	0:20	+0020
 Rule	Ghana	1920	1946	-	Jan	 1	2:00	0	GMT
-Rule	Ghana	1921	1939	-	Sep	 1	2:00	0:20	+0020
+Rule	Ghana	1920	1939	-	Sep	 1	2:00	0:20	+0020
 Rule	Ghana	1940	1945	-	May	 1	2:00	0:20	+0020
 Rule	Ghana	1950	1955	-	Sep	 1	2:00	0:30	+0030
 Rule	Ghana	1951	1956	-	Jan	 1	2:00	0	GMT
-- 
1.8.3.1

